bears/general/coalaBear.py

```diff
@@ -0,0 +1,24 @@
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+class coalaBear(LocalBear):
+    LANGUAGES = {"All"}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Spelling'}
+
+    def run(self, filename, file):
+        """
+        Check for the correct spelling of ``coala`` in the file.
+        """
+        for line_number, line in enumerate(file):
+            if line.find("Coala") != -1 or line.find('COALA') != -1:
```

Comment:
hm, could we rather do a regex match and match an upper case c and then oala case insensitive?

CC @Makman2